google/cloud/security/scanner/scanner.py

```diff
@@ -208,179 +149,48 @@ def _get_timestamp(global_configs, statuses=('SUCCESS', 'PARTIAL_SUCCESS')):
-    try:
-        vdao = violation_dao.ViolationDao(global_configs)
-        (inserted_row_count, violation_errors) = vdao.insert_violations(
-            all_violations,
-            resource_name=resource_name,
-            snapshot_timestamp=snapshot_timestamp)
-    except db_errors.MySQLError as err:
-        LOGGER.error('Error importing violations to database: %s', err)
-
-    # TODO: figure out what to do with the errors. For now, just log it.
-    LOGGER.debug('Inserted %s rows with %s errors',
-                 inserted_row_count, len(violation_errors))
-
-    # TODO: Remove this specific return when tying the scanner to the general
-    # violations table.
-    if resource_name != 'violations':
-        return
-
-    # Write the CSV for all the violations.
-    if scanner_configs.get('output_path'):
-        LOGGER.info('Writing violations to csv...')
-        output_csv_name = None
-        with csv_writer.write_csv(
-            resource_name=resource_name,
-            data=all_violations,
-            write_header=True) as csv_file:
-            output_csv_name = csv_file.name
-            LOGGER.info('CSV filename: %s', output_csv_name)
-
-            # Scanner timestamp for output file and email.
-            now_utc = datetime.utcnow()
-
-            output_path = scanner_configs.get('output_path')
-            if not output_path.startswith('gs://'):
-                if not os.path.exists(scanner_configs.get('output_path')):
-                    os.makedirs(output_path)
-                output_path = os.path.abspath(output_path)
-            _upload_csv(output_path, now_utc, output_csv_name)
-
-            # Send summary email.
-            if global_configs.get('email_recipient') is not None:
-                payload = {
-                    'email_sender': global_configs.get('email_sender'),
-                    'email_recipient': global_configs.get('email_recipient'),
-                    'sendgrid_api_key': global_configs.get('sendgrid_api_key'),
-                    'output_csv_name': output_csv_name,
-                    'output_filename': _get_output_filename(now_utc),
-                    'now_utc': now_utc,
-                    'all_violations': all_violations,
-                    'resource_counts': kwargs.get('resource_counts', {}),
-                    'violation_errors': violation_errors
-                }
-                message = {
-                    'status': 'scanner_done',
-                    'payload': payload
-                }
-                notifier.process(message)
-    # pylint: enable=too-many-locals
-
-def _upload_csv(output_path, now_utc, csv_name):
-    """Upload CSV to Cloud Storage.
+#    LOGGER.info('Initializing the rules engine:\nUsing rules: %s', FLAGS.rules)
 
-    Args:
-        output_path: The output path for the csv.
-        now_utc: The UTC timestamp of "now".
-        csv_name: The csv_name.
-    """
+    forseti_config = FLAGS.forseti_config
+    if forseti_config is None:
```

Comment:
Wouldn't this approach ignore FLAGS.list_engines and FLAGS.engine_name and just exit?